cohorts/load.py

```diff
@@ -1282,7 +1286,7 @@ def plot_joint(self, on, on_two=None, x_col=None, plot_type="jointplot", stat_fu
         jointplot_kwargs : dict, optional
```

Comment:
That seems like a good idea. Will make an issue for it unless you think it should happen in this PR?